yarn-project/p2p/src/msg_validators/tx_validator/metadata_validator.test.ts

```diff
@@ -94,24 +100,37 @@ describe('MetadataTxValidator', () => {
     await expectInvalid(badTxs[1], TX_ERROR_INCORRECT_PROTOCOL_CONTRACT_TREE_ROOT);
   });
 
-  it.each([42, 43])('allows txs with valid max block number', async maxBlockNumber => {
+  it.each([10n, 11n])('allows txs with valid expiration timestamp', async includeByTimestamp => {
     const [goodTx] = await makeTxs();
-    goodTx.data.rollupValidationRequests.maxBlockNumber = new MaxBlockNumber(true, maxBlockNumber);
+    goodTx.data.rollupValidationRequests.includeByTimestamp = new IncludeByTimestamp(true, includeByTimestamp);
 
     await expectValid(goodTx);
   });
 
-  it('allows txs with unset max block number', async () => {
+  it('allows txs with unset expiration timestamp', async () => {
     const [goodTx] = await makeTxs();
-    goodTx.data.rollupValidationRequests.maxBlockNumber = new MaxBlockNumber(false, 0);
+    goodTx.data.rollupValidationRequests.includeByTimestamp = new IncludeByTimestamp(false, 0n);
 
     await expectValid(goodTx);
   });
 
-  it('rejects txs with lower max block number', async () => {
+  it('rejects txs with lower expiration timestamp', async () => {
+    const [badTx] = await makeTxs();
+    badTx.data.rollupValidationRequests.includeByTimestamp = new IncludeByTimestamp(true, timestamp - 1n);
+
+    await expectInvalid(badTx, TX_ERROR_INVALID_INCLUDE_BY_TIMESTAMP);
+  });
+
+  it('accept txs with lower expiration timestamp when building block 1', async () => {
```

Comment:
Introduced this unit test to check that the check is indeed disabled at block 1.